__tests__/environment-test.js

```diff
@@ -0,0 +1,81 @@
+"use strict"
+
+jest.dontMock('../environment.js');
+
+describe('environment', () => {
+  let environment = require('../environment.js');
+
+  it('uses original config attributes if no env key is found', () => {
+    let parsedConfig = environment.checkEnvironmentForConfig({
```

Comment:
You should clear out the environment variable explicitly, otherwise it's confusing what this test is doing (nothing?)